public/app/features/dashboard/components/DashNav/DashNavTimeControls.tsx

```diff
@@ -114,7 +129,10 @@ class UnthemedDashNavTimeControls extends Component<Props> {
           onMoveBackward={this.onMoveBack}
           onMoveForward={this.onMoveForward}
           onZoom={this.onZoom}
+          onClose={this.onClose}
           onChangeTimeZone={this.onChangeTimeZone}
+          invalid={this.state.isError}
+          error={'Invalid time range'}
```

Comment:
I would like to see some more details about this error here, to guide user what is the configured allowed range

Reply:
Good idea, how does displaying the setting itself sound (e.g. Invalid time range: The maximum time back is 5h from now)? Also, would it make more sense to display the first violated rule or simply inform the user of both time limits?